tools/cldr-code/src/main/java/org/unicode/cldr/util/Annotations.java

```diff
@@ -216,7 +242,10 @@ public static final class AnnotationSet {
         static final Factory factory = CONFIG.getCldrFactory();
         static final CLDRFile ENGLISH = CONFIG.getEnglish();
         static final CLDRFile ENGLISH_ANNOTATIONS = null;
-        static final SubdivisionNames englishSubdivisionIdToName = new SubdivisionNames("en", "main");
```

Comment:
This is definitely splitting lines that it doesn't need to. Maybe set the width higher?

Reply:
I'll make another PR with printWidth 120

Reply:
See https://github.com/unicode-org/cldr/pull/1919 -- it leaves this line unchanged